main.tf

```diff
@@ -75,10 +75,13 @@ resource "azurerm_kubernetes_cluster" "aks" {
   api_server_authorized_ip_ranges = local.api_server_authorized_ip_ranges
 
   addon_profile {
```

Comment:
@dutsmiller this is the only piece of relevant code. the rest is `terraform format`